pkg/alicloud/apis/alicloud_provider_spec.go

```diff
@@ -52,6 +52,7 @@ type AlicloudDataDisk struct {
 
 // AlicloudSystemDisk describes SystemDisk for Alicloud.
 type AlicloudSystemDisk struct {
-	Category string `json:"category"`
-	Size     int    `json:"size"`
+	Category  string `json:"category"`
+	Size      int    `json:"size"`
+	Encrypted bool   `json:"encrypted,omitEmpty"`
```

Comment:
Don't we need core in the core creation logic to set the machine creation request based on this field?

Reply:
Yes, you are right. I closed this PR. Thanks.

Reply:
No problem. I see similar changes on in-tree code in the past [here](https://github.com/gardener/machine-controller-manager/pull/397/files#diff-0eeff75398ef946a5abbe96caaec69e8bed2f7602364d6113963f92d4989be9eR121). Are these changes present/missing in OOT?